src/EA.Weee.Web/Areas/Producer/Mappings/ToViewModel/OrganisationViewModelMap.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace EA.Weee.Web.Areas.Producer.Mappings.ToViewModel
+{
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Core.DirectRegistrant;
+    using EA.Weee.Core.Organisations;
+    using EA.Weee.Core.Organisations.Base;
+
+    public class OrganisationViewModelMap : IMap<SmallProducerSubmissionData, OrganisationViewModel>
+    {
+        public OrganisationViewModel Map(SmallProducerSubmissionData source)
+        {
+            return new OrganisationViewModel()
+            {
+                Address = new Core.Organisations.ExternalAddressData
+                {
```

Comment:
@seanciv There is actually a mapper for this ReverseAddressMap. So you can inject the mapper into the class and use it similar to EditOrganisationDetailsMap.